rpcs3/Emu/RSX/NV47/HW/common.cpp

```diff
@@ -72,6 +77,84 @@ namespace rsx
 			}
 		}
 
+		void set_texture_configuration_command(rsx::context* ctx, u32 reg)

```

Comment:
This is a restructure, not an optimization. I'd suggest moving this to a separate PR to avoid regressions.